exercises/sieve/instructions.md

```diff
@@ -5,24 +5,36 @@ Your task is to create a program that implements the Sieve of Eratosthenes algor
 A prime number is a number that is only divisible by 1 and itself.
 For example, 2, 3, 5, 7, 11, and 13 are prime numbers.
```

Comment:
They asked: why is 1 not listed as a prime number?

The answer is: so that we can declare that all numbers have a unique prime factorization. If 1 were prime, every number *n* would have lots of different prime factorizations: *n*, 1×*n*, 1×1×*n*, etc. But such an explanation does not fit into these instructions.

We could exclude 1 explicitly,

```suggestion
A prime number is a number larger than 1 that is only divisible by 1 and itself.
For example, 2, 3, 5, 7, 11, and 13 are prime numbers.
```

or we could exclude it implicitly, e.g.

```suggestion
A prime number is a number that is not divisible by any smaller number.
For example, 2, 3, 5, 7, 11, and 13 are prime numbers.
```

You can probably think of better expressions.